tests/foreman/sys/test_katello_certs_check.py

```diff
@@ -53,10 +53,16 @@ def setUpClass(cls):
         """Get host name, scripts, and create working directory."""
         super(KatelloCertsCheckTestCase, cls).setUpClass()
         _, cls.sat6_hostname = os.path.split(settings.server.hostname)
+        cls.capsule_hostname = 'capsule.example.com'
         cls.key_file_name = '{0}/{0}.key'.format(cls.sat6_hostname)
         cls.cert_file_name = '{0}/{0}.crt'.format(cls.sat6_hostname)
         cls.ca_bundle_file_name = 'cacert.crt'
         cls.SUCCESS_MSG = "Validation succeeded"
+        # Need a subdirectory under ssl-build with same name as Capsule name
+        with get_connection(timeout=100) as connection:
+            result = ssh.command('mkdir ssl-build/{0}'.format(cls.capsule_hostname))
```

Comment:
why not use ```connection.run``` here?

Reply:
Originally there was only one command there, so thats an over site on my part when I added the assert command. Fixed now.
Thank you